src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        I am
+        {` ${person.age}`}
+      </p>
+    )}
```

Comment:
```suggestion
    <h2 className="Person__name">
      {`My name is ${name}`}
    </h2>
    
    {person.age && (
      <p className="Person__age">
        {`I am ${age}`}
      </p>
    )}
```
Just use [interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here.
Also, let's do destructuring to get all the props of person
![image](https://user-images.githubusercontent.com/96409563/207810284-ce0dce44-d305-4569-898f-618dba411356.png)